unit-tests/dds/test-option-value.py

```diff
@@ -0,0 +1,116 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#test:donotrun:!dds
+
+import pyrealdds as dds
+from rspy import log, test
+
+# Test dds.option mechanism and logic
+
+with test.closure( 'read-only options' ):
+    test.check( dds.option.from_json( ['1', 0, 'desc'] ).is_read_only() )
+    test.check_false( dds.option.from_json( ['1', 0, 0, 'desc'] ).is_read_only() )  # default value -> not read-only
+    test.check( dds.option.from_json(
+        ['Asic Temperature',None,-40.0,125.0,0.0,0.0,'Current Asic Temperature (degree celsius)',['optional','read-only']] ).is_read_only() )
+    test.check_equal_lists( dds.option.from_json(  # default==min==max, step==0 -> read-only!
+        ['Stereo Baseline',50.20994567871094,50.20994567871094,50.20994567871094,0.0,50.20994567871094,'...',['read-only']] ).to_json(),
```

Comment:
Doesn't read only supposed to be implied with default==min==max, step ==0? You already checked it with the properties

Reply:
Yes, it is implied - this checks it.

Reply:
So you should remove the `[read=only]` property here. I am not sure it is implied and not taken from the property first.